src/scss/components/ec-card/_ec-card.scss

```diff
@@ -4,7 +4,35 @@
 
   padding: 8px;
   background: $white;
-  overflow: hidden;
+  word-break: break-word;
+
+  &--is-interactive {
+    position: relative;
+    cursor: pointer;
+    transition: all 0.4s ease;
+
+    &:after {
+      @include box-shadow-level-2;
+
+      content: '';
+      position: absolute;
+      z-index: -1;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      opacity: 0;
+      transition: all 0.4s ease;
```

Comment:
Same here. Plus you repeated `0.4s ease` 2 times, I'd extract it into a variable (e.g. `ec-card-interactive-transition`) because the transitions should be in sync, right? Do we expect having different transition for opacity and different for transform?

Reply:
@mcibique At first, we expect to have the same, done, thanks!